OpenOversight/app/templates/base.html

```diff
@@ -143,56 +149,42 @@
+            <a class="nav-link mx-4" href="{{ url_for('main.submit_data') }}">Submit Images</a>
+            <a class="nav-link mx-4"
+               href="{{ url_for('main.get_started_labeling') }}">Volunteer</a>
+            <a class="nav-link mx-4" href="{{ url_for('main.about_oo') }}">About</a>
+            <a class="nav-link mx-4" href="{{ url_for('main.contact_oo') }}">Contact</a>
           </div>
-          <div class="col-sm-4">
-            <h5>Contact</h5>
-            <a title="TechBloc Twitter"
-               href="https://twitter.com/TechBlocSEA"
-               rel="noopener noreferrer"
-               target="_blank"><i class="fa-brands fa-square-x-twitter fa-3x social" aria-hidden="true"></i></a>
-            <a title="OpenOversight GitHub"
-               href="https://github.com/OrcaCollective/OpenOversight"
-               rel="noopener noreferrer"
-               target="_blank"><i class="fa-brands fa-github-square fa-3x social" aria-hidden="true"></i></a>
-            <a href="mailto:techblocsea@protonmail.com"><i class="fa-solid fa-envelope-square fa-3x social" aria-hidden="true"></i></a>
-            <a title="TechBloc Projects"
-               href="https://tech-bloc-sea.dev/"
-               rel="noopener noreferrer"
-               target="_blank"><i class="fa-solid fa-globe fa-3x social" aria-hidden="true"></i></a>
-            <br>
-            <a href="{{ url_for('main.privacy_oo') }}" class="btn">Privacy Policy</a>
-          </div>
-          <div class="col-sm-4">
-            <h5>Navigation</h5>
-            <p class="font-weight-300">
-              <a href="{{ url_for('main.get_officer') }}">Find an Officer</a>
-              <br>
-              <a href="{{ url_for('main.submit_data') }}">Submit Images</a>
-              <br>
-              <a href="{{ url_for('main.get_started_labeling') }}">Volunteer</a>
-              <br>
-              <a href="{{ url_for('main.about_oo') }}">About</a>
+        </div>
+        <div class="row text-center horizontal-padding">
+          <div class="col-sm-12">
+            <p class="font-weight-300">spd.watch is independently maintained by Seattle Tech Bloc.</p>
+            <a href="https://lucyparsonslabs.com/" target="_blank">
+              <img src="{{ url_for('static', filename='images/lpl-logo-white-transparent.png') }}"
+                   class="mt-4 lpl-logo"
+                   alt="Lucy Parsons Labs logo">
+            </a>
+            <p>
+              <a href="https://bsky.app/profile/divestspd.bsky.social"><i class="fa-brands fa-bluesky fa-3x social"></i></a>
+              <a href="https://instagram.com/divestspd"><i class="fa-brands fa-instagram fa-3x social"></i></a>
+              <a href="https://github.com/OrcaCollective/openoversight"><i class="fa-brands fa-github fa-3x social"></i></a>
+              <a href="mailto:techblocsea@protonmail.com"><i class="fa fa-envelope fa-3x social"></i></a>
+            </p>
```

Comment:
Let me know if there's anything that needs to be updated in this section

Reply:
I think this is good!